ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+Make sure you try the following before submitting the issue, thank you!
+
+- Check that umake is the latest version (16.11.1)
+- If you're not running the latest development version of Ubuntu, zesty (17.04), or a distro based on zesty, add the ppa as described in the README.
+
+If all this doesn't solve the problem you're having, please submit the issue including the version of ubuntu and umake.
```

Comment:
what I mean is for instance:
Ubuntu Mate 16.10
umake 16.11.1

So both ubuntu and umake